### PR TITLE
Adds dagman_progress command line tool

### LINKED ISSUE #44: Add job monitoring utility
#### Description

I'd like to have some kind of dagman monitoring functionality &mdash; a progress bar would be a good start!

### PR DESCRIPTION
**Pull request by jrbourbeau**
#### Reference Issue

Fixes #44

#### What does this pull request implement/fix? Explain your changes.

This PR adds a command line tool, `dagman_progress`, that prints a progress bar for submitted `pycondor.Dagman` objects. 

Example use, 
```bash
$ dagman_progress /path/to/dagman/submit_file.submit
[##############################] | 100% complete (14 done, 0 queued, 0 ready, 0 unready, 0 failed)
```

### CODE REVIEW

**Comment by jrbourbeau** [comment]
@dsschult what do you think?

**Comment by dsschult** [comment]
I prefer opening the dagman out file once, instead of multiple times:
https://stackoverflow.com/questions/12523044/how-can-i-tail-a-log-file-in-python/12523119#12523119
Raymond Hettinger is a python dev, so his solution to tailing a file is obviously good.

**Comment by jrbourbeau** [comment]
Thanks for the comment @dsschult. I updated `dagman_progress` to only read in the log file once. I also switched to a generator-based method for parsing based on the stackoverflow link you sent.

**Comment by dsschult** [comment]
Aren't generators great? :)

**Comment by jrbourbeau** [comment]
They're pretty awesome. I'm gonna have to start using them more :tada:

File: pycondor/command_line.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    if node_counts:
+        status = Status(*node_counts)
+    else:
+        status = Status(*[0]*len(_states))
+
+    return status
+
+
+def status_genreator(dag_out_file):
```
Comment (kreczko) [inline pycondor/command_line.py]: Did you mean `status_generator`?

Comment (jrbourbeau) [reply pycondor/command_line.py]: That's definitely what I mean. Good catch @kreczko, thanks!

**Pull request merged**